### ISSUE TITLE
How to make the bar layer under the boss bar?

### ISSUE THREAD

**Issue by betterworld-sjq** [jahirxtrap/healthindicatortxf#4]
![image](https://github.com/jahirxtrap/healthindicatortxf/assets/140177869/0ceb4490-604a-4794-b8fe-79849b68b1bb)
This guy is a datapack activated boss so he isn't categorized as a boss to hide the bar. So I hope the red modded bar could be covered by the vanilla bar(the blue one)

**Comment by kuhananxianglai** [comment]
Yes, I also encountered such a problem, in another mod "Aquamirae", there is a captain Boss, which is not correctly recognized as a Boss mob to hide the HUD.
In fact, it is unlikely that we will take the initiative to be compatible with these Boss-containing mods one by one, so my suggestion is to add a custom function like a Boss category list.
Currently, in the mod’s configuration file, there is only a simple entry for whether the Boss is disabled or not, and a Boss collection list that can be customized by the player should be added.
All mobs in this list should have the HUD hidden, the default mobs should be Ender Dragon and Wither.
![2024-08-23_21 48 31](https://github.com/user-attachments/assets/82bb3da6-9203-4ee9-885d-008a4d72d444)
![屏幕截图 2024-08-23 215755](https://github.com/user-attachments/assets/9f043e14-aaaa-4356-9fa0-25823962cd5e)

**Comment by kuhananxianglai** [comment]
Actually, I don't think there's any need to show extra health information in such a tense scene, and vanilla's Boss bar is sufficient.
So take it a step further, we can completely disable the HUD when fighting entities in the boss list. 
This effectively avoids the problem such as displaying the enderman's HUD when fighting the Ender Dragon. 
Yes, even though the Ender Dragon's HUD is hidden, it still shows the enderman's HUD and obscures vanilla's Boss bar.
This has the added benefit of being compatible with all mod's bosses and all related resource packs with boss bar textures.
Of course, this problem only occurs in players who display the HUD layout at the top center of the screen, as this will obscure the vanilla's Boss bar, and hopefully this problem can be solved as soon as possible:)
Oh, I almost forgot one thing, there's also a mod called "Jade", maybe we can refer to how it solves this problem.
